cr17/nodes/cr17/high_level_state_controller.py

```diff
@@ -36,6 +37,9 @@ class high_level_state_controller(object):
     def __init__(self):
         rospy.init_node("high_level_state_controller")
 
+        self.state_pub = rospy.Publisher(STATE_TOPIC, String, queue_size=10)
+        self.puber = rospy.Publisher('/home', Bool, queue_size=10)
```

Comment:
Agreed. It is subscribed to by `path_pub.py` and is being used to toggle between an outgoing and incoming path. I _think_ this is a temporary topic because `path_pub.py` is a temporary node that will be replaced by `path_planner.py`